### ISSUE TITLE
PPO Critic

### ISSUE THREAD

**Issue by MagMueller** [leikun-starting/End-to-end-DRL-for-FJSP#8]
@leikun-starting Thank you very much!

I have some questions in regard to the critic, maybe from there some improvements arise. 

- Currently the critic only receives the job embedding and not the machine embedding
- Why are you not updating the critic separately, e.g. with the job_v_loss 
`job_v_loss = self.MSE(val[j], rewards_all_env[j])`
currently you simply add the critic loss to the actor loss.
- The critic is used in both loss functions (job and machine)
